### PR TITLE
task/FP-1855 Completed Submissions Page Refactor

### PR DESCRIPTION
**Pull request by shayanaijaz**
## Overview

Refactor based off the previously merged in [task/FP-1855 Submissions List Page](https://github.com/TACC/Core-CMS-Custom/pull/63) PR

## Related

[FP-1885 - My (Completed) Submissions page](https://jira.tacc.utexas.edu/browse/FP-1885)

## Changes

1. Removed `Created` column from table and added `Received` column
2. Adjusted `File Name` column sizing
3. Adjusted static text on the page
4. Adjusted modal layout to display label and value on the same line

## Testing

1. In order to mock some data for local testing make the following changes in `apcd_cms/src/apps/submissions/views.py`:

<details>
       <summary>View Snippet</summary>

Add the following to the top of the file:

`import datetime`

 Replace line 38 with the following:

  ```js
submission_content = [
      (
          1,                                                      # submission_id
          1,                                                      # apcd_id
          1,                                                      # submitter_id
          'T_GLDNRLE_10000000_202303_202303.zip',                 # zip_file_name
          datetime.datetime(2022, 9, 27, 15, 25, 24, 285442),     # received_timestamp
          202303,                                                 # data_period_start
          202303,                                                 # data_period_end
          'T',                                                    # test_submission_flag
          'complete',                                             # status
          'failed',                                               # outcome
          'great expectations validation failed',                 # outcome_reason
          0,                                                      # compressed_size
          None,                                                   # processed_timestamp
          None,                                                   # created_at
          None,                                                   # updated_at
          None                                                    # notes
      ),
      (
          2,                                                      # submission_id
          1,                                                      # apcd_id
          1,                                                      # submitter_id
          'T_GLDNRLE_10000000_202304_202304.zip',                 # zip_file_name
          datetime.datetime(2022, 9, 29, 10, 2, 22, 276789),      # received_timestamp
          202304,                                                  # data_period_start
          202304,                                                 # data_period_end
          'T',                                                    # test_submission_flag
          'complete',                                             # status
          'failed',                                               # outcome
          'great expectations validation failed',                 # outcome_reason
          0,                                                      # compressed_size
          None,                                                   # processed_timestamp
          None,                                                   # created_at
          None,                                                   # updated_at
          None                                                    # notes
      ),
      (
          3,                                                      # submission_id
          1,                                                      # apcd_id
          1,                                                      # submitter_id
          'T_GLDNRLE_10000000_202301_202301.zip',                 # zip_file_name
          datetime.datetime(2022, 9, 27, 15, 25, 24, 285442),     # received_timestamp
          202303,                                                 # data_period_start
          202303,                                                 # data_period_end
          'T',                                                    # test_submission_flag
          'in process',                                           # status
          None,                                                   # outcome
          None,                                                   # outcome_reason
          0,                                                      # compressed_size
          None,                                                   # processed_timestamp
          None,                                                   # created_at
          None,                                                   # updated_at
          None                                                    # notes
      ),
      (
          4,                                                      # submission_id
          1,                                                      # apcd_id
          1,                                                      # submitter_id
          'T_GLDNRLE_10000000_202303_202303.zip',                 # zip_file_name
          datetime.datetime(2022, 9, 27, 15, 25, 24, 285442),     # received_timestamp
          202303,                                                 # data_period_start
          202303,                                                 # data_period_end
          'T',                                                    # test_submission_flag
          'in process',                                           # status
          None,                                                   # outcome
          None,                                                   # outcome_reason
          0,                                                      # compressed_size
          None,                                                   # processed_timestamp
          None,                                                   # created_at
          None,                                                   # updated_at
          None                                                    # notes
      )
  ]
  ```

Replace line 70 with the following: 

  ```js
submission_logs = [
        (
            1,                          # log_id
            1,                          # submission_id
            '  ',                       # file_type
            'init_txt_file_suite',      # validation_suite
            {'test': 'here'},           # json_long
            'failed'                    # outcome
        ),
        (
            2,                          # log_id
            1,                          # submission_id
            'ME',                       # file_type
            'eligibility_suite',      # validation_suite
            {'test': 'here'},           # json_long
            'failed'  
        ),
        (
            3,                          # log_id
            1,                          # submission_id
            'PC',                       # file_type
            'pharmacy_suite',      # validation_suite
            {'test': 'here'},           # json_long
            'failed'  
        ),
        (
            4,                          # log_id
            1,                          # submission_id
            'PV',                       # file_type
            'provider_suite',      # validation_suite
            {'test': 'here'},           # json_long
            'failed'  
        )
    ]
  ```
</details>

2. Open [http://localhost:8000/submissions/list-submissions](http://localhost:8000/submissions/list-submissions)
3. Ensure data is displayed on the page in a table layout with the updated columns
4. Ensure a modal opens when clicking the `View Logs` button with information about the logs

## UI

**Before** 

Screen Shot 2023-01-13 at 1 13 03 PM

Screen Shot 2023-01-13 at 1 13 13 PM

**After**

Screen Shot 2023-01-18 at 3 19 29 PM

Screen Shot 2023-01-18 at 2 11 00 PM

### CODE REVIEW

File: apcd-cms/src/apps/submissions/templates/view_submission_logs_modal.html [hunk 1]
```diff
@@ -12,23 +12,21 @@ <h4 class="modal-title">View Logs</h4>
...
-                    <dd class="c-data-list__value">{{log.submitter_id}}</dd>                       
-                    <dt class="c-data-list__key">File Type</dt>                
-                    <dd class="c-data-list__value">{{log.file_type}}</dd>
-                    <dt class="c-data-list__key">Validation Suite</dt>                
-                    <dd class="c-data-list__value">{{log.validation_suite}}</dd>
-                    <dt class="c-data-list__key">Outcome</dt>                
-                    <dd class="c-data-list__value">{{log.outcome}}</dd>
+                        <dt class="c-data-list__key">Log ID</dt>
+                        <dd class="c-data-list__value">{{log.log_id}}</dd>                    
+                        <dt class="c-data-list__key">File Type</dt>
```
Comment (sophia-massie) [inline apcd-cms/src/apps/submissions/templates/view_submission_logs_modal.html:21]: It looks like Submitter ID is no longer listed in the modal. Was that requested to be removed and this is intentional? Just double checking with the screen shot and my modal only has Log ID, file type, validation suite, and outcome listed per log.

Comment (shayanaijaz) [reply apcd-cms/src/apps/submissions/templates/view_submission_logs_modal.html:21]: Sorry I attached an older screenshot. Yes Tracy asked me to remove that field

**Review by sophia-massie — APPROVED** [review]
LGTM

**Review by edmondsgarrett — APPROVED** [review]
LGTM!

**Pull request merged**